holepunch/__init__.py

```diff
@@ -315,7 +329,15 @@ def holepunch(args):
 
     command = args['--command']
     if command is not None:
-        print('Rules will revert when `%s` terminates.' % command)
+        log('Rules will revert when `%s` terminates.' % command)
+
+        # noinspection PyUnusedLocal
+        def terminate(num, frame):
```

Comment:
let's move this next to the `atexit.register` call so all the cleanup logic is in one place.